build.gradle

```diff
@@ -5,14 +5,14 @@ buildscript {
       mavenCentral()
   }
   dependencies {
-      classpath 'com.android.tools.build:gradle:1.1.0'
-      classpath 'org.apache.commons:commons-io:1.3.2'
-      classpath 'com.github.dcendents:android-maven-plugin:1.2'
+      classpath 'com.android.tools.build:gradle:1.5.+'
+      classpath 'commons-io:commons-io:2.4'
+      classpath "com.github.dcendents:android-maven-gradle-plugin:1.3"
```

Comment:
Please keep single quotes for consistency, check & fix other places in this diff too please.